experimental/packages/sdk-events/README.md

```diff
@@ -0,0 +1,79 @@
+# OpenTelemetry Events SDK
+
+[![NPM Published Version][npm-img]][npm-url]
+[![Apache License][license-image]][license-image]
+
+**Note: This is an experimental package under active development. New releases may include breaking changes.**
+
+This package provides an experimental Events SDK implementation. The Events SDK is implemented on top of the Logs SDK.
+
+## Installation
+
+```bash
+npm install --save @opentelemetry/api-logs
+npm install --save @opentelemetry/api-events
+npm install --save @opentelemetry/sdk-events
+```
+
+## Usage
+
+Here is an example of configuring and using the Events SDK:
+
+```js
+const { logs } = require('@opentelemetry/api-logs');
+const { events } = require('@opentelemetry/api-events');
+const { EventLoggerProvider } = require('@opentelemetry/sdk-events');
+const {
+  LoggerProvider,
+  SimpleLogRecordProcessor,
+  ConsoleLogRecordExporter,
+} = require('@opentelemetry/sdk-logs');
+
+// The Events SDK has a dependency on the Logs SDK.
+// Any processing of events (e.g. export) is done through the Logs SDK.
+const loggerProvider = new LoggerProvider();
+loggerProvider.addLogRecordProcessor(
+  new SimpleLogRecordProcessor(new ConsoleLogRecordExporter())
+);
+
+// Register a global EventLoggerProvider.
+// This would be used by instrumentations, similar to how the global TracerProvider,
+// LoggerProvider and MeterProvider work.
+const eventLoggerProvider = new EventLoggerProvider(loggerProvider);
+events.setGlobalEventLoggerProvider(eventLoggerProvider);
+
+// Get an EventLogger from the global EventLoggerProvider
+const eventLogger = events.getEventLogger('default');
+
+// Emit an event
+eventLogger.emit({
+  name: 'myEvent',
```

Comment:
```suggestion
  name: 'myDomain.myEvent',
```
Similar to [Santosh's comment](https://github.com/open-telemetry/opentelemetry-js/pull/4629/files#r1610752698) on the example app, I like the idea of adding a domain here to encourage its use. Also I realize this example is camelCase and the other is kebab-case. Do we have a suggestion/recommendation? This may be pedantic but a consistent example naming scheme might be useful here as folks start understanding events.